### PR TITLE
[DOCS] Document `max_concurrent_searches` default

### LINKED ISSUE #55033: [DOCS] Multi Search - Default value for `max_concurrent_searches` not documented
On the documentation page for [Multi Search](https://www.elastic.co/guide/en/elasticsearch/reference/7.6/search-multi-search.html), the default value for `max_concurrent_searches` is not specified.

It took a bit of asking on Slack and digging through the code to figure it out. Found it [here](https://github.com/elastic/elasticsearch/blob/2843992b1168f585349cfb7d1902e2aacc558cd6/server/src/main/java/org/elasticsearch/action/search/TransportMultiSearchAction.java#L107) in the end. It would be great to have this in the docs.

### PR DESCRIPTION
**Pull request by jrodewig**
Documents the default value for the multi search API's `max_concurrent_searches`
query parameter.

Closes #55033

### CODE REVIEW

File: docs/reference/search/multi-search.asciidoc [hunk 1]
```diff
@@ -78,7 +78,8 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=index-ignore-unavailable]
 
 `max_concurrent_searches`::
 (Optional, integer)
-Maximum number of concurrent searches the multi search API can execute.
+Maximum number of concurrent searches the multi search API can execute. Defaults
+to +max(1, (# of <<data-node,data nodes>> * min(# of <<node.processors,allocated processors>>, 10)))+.
```
Comment (martijnvg) [inline docs/reference/search/multi-search.asciidoc:82]: This part: `<<node.processors,allocated processors>>` is more like: `((<<node.processors,allocated processors>> * 3) / 2) + 1`. But perhaps we just refer to the search thread default size?

Comment (jrodewig) [reply docs/reference/search/multi-search.asciidoc]: Good catch. I've replaced `<<node.processors,allocated processors>>` with `<<search-threadpool,search thread pool size>>` and added an appropriate anchor to the search thread pool definition.

Thanks @martijnvg!

**Review by martijnvg — APPROVED** [review]
LGTM

**Comment by jrodewig** [comment]
### Backport commits
master 4980ea7
7.x 8d6f0f6a769
7.7 4fa40a0750e
7.6 dd5b4d2f723795f3d59c16c3c145119ee60fb64e

**Pull request merged**